labelImg.py

```diff
@@ -204,141 +206,145 @@ def __init__(self, default_filename=None, default_prefdef_class_file=None, defau
 
         # Actions
         action = partial(new_action, self)
-        quit = action(get_str('quit'), self.close,
```

Comment:
Rationale: Avoid using reserved names as variables. 
For consistency I prepended "action_" to all action variables